functests/4_latency/latency.go

```diff
@@ -61,8 +61,11 @@ var _ = Describe("[performance] Latency Test", func() {
 	var profile *performancev2.PerformanceProfile
 	var latencyTestPod *corev1.Pod
 	var err error
+	var logName string
 
 	BeforeEach(func() {
+		logName = time.Now().Format("20060102150405")
```

Comment:
You will have the same ```logName``` for all the tests.
So in case that more than one tool is running, the log file of the first tool will be overridden by the log file of the second tool.

Reply:
@Tal-or  logName will be different for each test , since it is initialized in "BeforeEach" & will be deleted at the end of each test in "afterEach"

Reply:
I'm still not fond of this way of creating log filenames, but we can improve in a later PR.